tests/integration-tests/tests/createami/test_createami/test_build_image/image.config.yaml

```diff
@@ -14,17 +14,14 @@ Build:
     Tags:
         - Key: dummyBuildTag
           Value: dummyBuildTag
-{% if os in ["alinux2", "alinux2023"] %}
-# Use UpdateOSPackages for alinux is optional. Use UpdateOSPackages on Ubuntu, RHEL/Rocky is not recommended because it intermittently fails because of kernel issues.
     UpdateOsPackages:
-        Enabled: true
-{% endif %}
-{% if os in ["ubuntu2204", "rhel9", "rocky9"] %}
-# Disable Lustre installation because these newer operating systems release new kernels more often. Lustre usually does not support the latest kernels
+        Enabled: {{ update_os_packages }}
     Installation:
         LustreClient:
-            Enabled: false
-{% endif %}
+            # Disable Lustre installation because these newer operating systems release new kernels more often. Lustre usually does not support the latest kernels
+            Enabled: {% if os in ["ubuntu2204", "rhel9", "rocky9"] %} false {% else %} true {% endif %}
```

Comment:
i wonder if we even care about 9.5 until this is supported.

Reply:
I agree. Anyway it is good to make the tests more robust

Reply:
lustre should just always be enabled i think

Reply:
Done, added another commit